web/src/invite.ts

```diff
@@ -389,8 +387,8 @@ function open_invite_user_modal(e: JQuery.ClickEvent<Document, undefined>): void
                 $button.text($t({defaultMessage: "Invite"}));
                 $button.attr("data-loading-text", $t({defaultMessage: "Inviting..."}));
             } else {
-                $button.text($t({defaultMessage: "Generate invite link"}));
-                $button.attr("data-loading-text", $t({defaultMessage: "Generating link..."}));
+                $button.text($t({defaultMessage: "Create link"}));
+                $button.attr("data-loading-text", $t({defaultMessage: "Creating link..."}));
```

Comment:
We should change the literal `...` to the unicode character; it looks better. Otherwise lgtm.